ngraph/src/ngraph/op/prior_box.cpp

```diff
@@ -71,7 +71,9 @@ void op::PriorBox::validate_and_infer_types()
 
         set_output_type(0,
                         element::f32,
-                        Shape{2, 4 * layer_shape[0] * layer_shape[1] * number_of_priors(m_attrs)});
+                        Shape{2,
```

Comment:
How it relates to this PR?

Reply:
not related to this PR. will be moved to https://github.com/openvinotoolkit/openvino/pull/1032/